### PR TITLE
Add installation script for Linux

### LINKED ISSUE #73: Script for installing withExEditorHost
I remember that when first using withExEditor, installing the corresponding host wasn't too easy.

Since then I've automated installation using the following script. It downloads the host from GitHub using cURL, extracts it with 7z and points `withexeditorhost.sh` to the host's index file:

    #! /usr/bin/env bash

    # Install the host for withExEditor that allows editing text in Firefox using a text editor like Vim.
    # After executing this script, reload the Firefox plugin
    withExEditorHostRemoteFile="https://github.com/asamuzaK/withExEditorHost/releases/download/v5.3.0-b.2/linux-x86_64.zip"
    withExEditorHostLocalZipFile="/tmp/withExEditorHost.zip"
    withExEditorHostDir="$HOME/.local/bin/withExEditorHost"

    # Create the dir for the host's index file, download the archive and unzip it
    mkdir --parents "$withExEditorHostDir"; curl -L -o "$withExEditorHostLocalZipFile" "$withExEditorHostRemoteFile" && 7z x "$withExEditorHostLocalZipFile" -o"$withExEditorHostDir"

    indexFile="$withExEditorHostDir/index"
    hostScript="$HOME/.config/withexeditorhost/config/firefox/withexeditorhost.sh"

    # The Firefox plugin will use this shell script to call the host's index file
    printf "#! /usr/bin/env bash\n'$indexFile'\n" > "$hostScript"

    chmod +x "$indexFile" "$hostScript"

I'm posting it here for three reasons:

  * For other users to benefit
  * You, asamuzaK, might want to include it as part of your installation instructions
  * Perhaps somebody knows how to not hardcode the version number in `withExEditorHostRemoteFile`, maybe replacing it with a URL of the latest release

### PR DESCRIPTION
**Pull request by mb720**
Fix #73
Using your suggestion for getting the latest version number of the host, I've enhanced the automation script for Linux.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -111,39 +111,39 @@ Enable withExEditor after the upgrade.
...
 
-    indexFile="$withExEditorHostDir/index"
-    hostScript="$HOME/.config/withexeditorhost/config/firefox/withexeditorhost.sh"
-
-    # The Firefox plugin will use this shell script to call the host's index file
-    printf "#! /usr/bin/env bash\n'$indexFile'\n" > "$hostScript"
-
-    chmod +x "$indexFile" "$hostScript"
+indexFile="${withExEditorHostDir}/index"
+hostScript=""${HOME}"/.config/withexeditorhost/config/firefox/withexeditorhost.sh"
```
Comment (asamuzaK) [inline README.md]: Just a question.
Are these quotes around `"${HOME}"` needed?

Comment (mb720) [reply README.md]: I added the quotes in case `$HOME` contains spaces. I'm not sure if that's allowed in Linux.

Comment (asamuzaK) [reply README.md]: There are quotes at the beginning and the end, so `"${HOME}/.config/withexeditorhost/config/firefox/withexeditorhost.sh"` should be treated as a single string, I think.

Comment (asamuzaK) [reply README.md]: To me, current implement seems to be parsed as:
[empty string] [unquoted ${HOME}] [quoted .config/...]

Comment (mb720) [reply README.md]: Makes sense. I removed those quotes.

**Comment by asamuzaK** [comment]
Merged.
Thanks a lot!

**Comment by mb720** [comment]
A pleasure. Thanks for your input! :-)

**Pull request merged**